client-core/src/synchronizer/auto_sync.rs

```diff
@@ -71,6 +73,21 @@ impl AutoSync {
         self.send_json(data)
     }
 
+    /// Get sync information
+    pub fn sync_info(&self) -> Result<AutoSyncInfo> {
```

Comment:
`-> AutoSyncInfo`

it seems It won't fail (if the lock is poisoned, it'll panic which is ok)

Reply:
ok